sensors/ev3_uart_sensor_defs.c

```diff
@@ -46,7 +46,7 @@ const struct ev3_uart_sensor_info ev3_uart_sensor_defs[] = {
 			},
 			[1] = {
 				/**
-				 * @description: Ambient light<br />Red LEDs off
+				 * @description: Ambient light<br />Blue LEDs on
```

Comment:
It is probably worth noting here that the blue LED is not at full intensity, but is rather dimly lit.